packages/app-project/stores/Project.spec.js

```diff
@@ -0,0 +1,77 @@
+import Project, { defaultProjectState } from './Project'
+import nock from 'nock'
+
+// Required until we swap out axios for the panoptesjs transport lib
+import axios from 'axios'
+import httpAdapter from 'axios/lib/adapters/http'
+axios.defaults.adapter = httpAdapter
+
+describe('Stores > Project', function () {
+  it('should export an object', function () {
+    Project.should.be.an('object')
+  })
+
+  describe('model', function () {
+    describe('data property', function () {
+      it('should exist', function () {
+        const store = Project.create(defaultProjectState)
+        store.data.should.be.an('object')
+      })
+    })
+
+    describe('isLoading property', function () {
+      it('should exist', function () {
+        const store = Project.create(defaultProjectState)
+        store.isLoading.should.be.a('boolean')
+      })
+    })
+  })
+
+  describe('actions', function () {
+    describe('fetch', function () {
+      it('should exist', function () {
+        const store = Project.create(defaultProjectState)
+        store.fetch.should.be.a('function')
+      })
+
+      it('should correctly request a project when passed a slug', function (done) {
+        const data = {
+          projects: [{
+            baz: 'beep'
+          }]
+        }
+
+        // Here we introduce a delay before the response ends. This allows us
+        // to test whether the store.isLoading property is updating correctly.
+        nock('https://www.zooniverse.org')
```

Comment:
We already talked about this, but once panoptes.js is brought in, the app shouldn't need to concern itself with mocking and testing API requests.

Reply:
🎉 this is already on the way in #29